actions/mongodb_query_admincommand_action_test.go

```diff
@@ -73,4 +73,37 @@ func TestMongoDBBuildinfo(t *testing.T) {
 		assert.Equal(t, "mozjs", m.Get("javascriptEngine").Data())
 		assert.Equal(t, "x86_64", m.Get("buildEnvironment.distarch").Data())
 	})
+
+	t.Run("getCmdLineOpts", func(t *testing.T) {
+		a := NewMongoDBQueryAdmincommandAction("", tests.GetTestMongoDBDSN(t), "getCmdLineOpts", 1)
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+		defer cancel()
+
+		b, err := a.Run(ctx)
+		require.NoError(t, err)
+
+		data, err := agentpb.UnmarshalActionQueryResult(b)
+		require.NoError(t, err)
+		assert.Len(t, data, 1)
+		t.Log(spew.Sdump(data))
+
+		m := objx.Map(data[0])
+
+		argv := m.Get("argv").InterSlice()
+		for _, v := range []interface{}{"mongod", "--profile", "2", "--auth"} {
```

Comment:
We can do different expected arguments for each db version like we do for PG, but I'm ok with it because it looks enough for this test.